<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>411</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLineEdit" name="newPassword">
      <property name="text">
       <string>변경할 비밀번호</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="newPasswordRe">
      <property name="text">
       <string>변경할 비밀번호 재입력</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="currentPassword">
      <property name="text">
       <string>현재 비밀번호</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_change">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>변경하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
